modules/beamdyn/tests/test_BD_ComputeIniNodalCrv.F90

```diff
@@ -27,7 +27,7 @@ subroutine test_BD_ComputeIniNodalCrv()
     
     ! --------------------------------------------------------------------------
     testname = "Tangent aligned with z-axis and 0 degree twist:"
-    n = (/ 0.0_BDKi, 0.0_BDKi, 1.0_BDKi /) ! tangent axis
+    n = (/ real(0.0, BDKi), real(0.0, BDKi), real(1.0, BDKi) /) ! tangent axis
```

Comment:
I'm surprised there were any issues with the original code for this line.  I can see other lines were missing the `_BDKi` part, but I'm surprised that writing `0.0_BDKi` was problematic.